blades/blade_base.h

```diff
@@ -16,6 +16,21 @@ enum BladeEffectType {
   EFFECT_IGNITION = 1 << 9,
   EFFECT_RETRACTION = 1 << 10,
   EFFECT_CHANGE = 1 << 11, // used for click to change
+
+  // Blaster Effect Types
+  EFFECT_STUN = 1 << 12, // used to pass STUN to allow alternate colors for stun type blasts
```

Comment:
Lots of bits, but I suppose we could potentially share them between props if needed, kind of like we share bits between D-pads and blaster buttons.

Reply:
Do you have a preference as to which I overlap with?

Reply:
No need to do any overlap yet.
I think we'll cross that bridge when we come to it.

Reply:
Ok